spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -2,10 +2,45 @@
 module Api
   module V1
     describe UsersController do
-      describe 'create' do
-        it 'throws a 501 error' do
-          post :create
-          expect(response.status).to eq(501)
+      describe 'POST #create' do
+        context 'when creating a new user' do
+          it 'creates the user successfully' do
+            post :create, params: { email: 'test@example.com' }
+
+            expect(response).to have_http_status(:created)
```

Comment:
this is nitpicking but: technically you should proabbly put the `post` in a `before :each` block and have the 3 expectations be separate specs, since each spec is supposed to test only 1 behavior